content/docs/started/getting-started-gke.md

```diff
@@ -4,6 +4,14 @@ description = "Get Kubeflow running on Google Cloud Platform (GCP)"
 weight = 2
 +++
 
-Please refer to the [deployment section](/docs/gke/deploy/) of the 
-[Kubeflow GCP documentation](/docs/gke/) for information on setting up your GCP 
-environment and deploying Kubeflow on Kubernetes Engine (GKE).
+For details on setting up your GCP
+environment and deploying Kubeflow on Kubernetes Engine (GKE),
+refer to the [deployment section](/docs/gke/deploy/) of the
+[Kubeflow GCP documentation](/docs/gke/).
+
+Alternatively, view this walkthrough of installing Kubeflow on GCP via
```

Comment:
Suggestion:

Alternately, view the following walk-through which demonstrates how to install Kubeflow on GCP using
 both the UI and CLI. All code used in this video can be found in the
[examples
repository](https://github.com/kubeflow/examples/tree/master/videos/from_zero_to_kubeflow).